hypha/apply/funds/templates/funds/includes/admin_primary_actions.html

```diff
@@ -15,123 +15,141 @@ <h5>{% trans "Actions to take" %}</h5>
         {% trans "Unarchive Submission" %}
     </a>
 {% else %}
-    {% if PROJECTS_ENABLED %}
-        {% if object.accepted_for_funding and not object.project %}
-            <button class="button button--bottom-space button--primary button--full-width"  hx-get="{% url 'funds:submissions:create_project' pk=object.pk %}" hx-target="#htmx-modal">
-                {% trans "Create Project" %}
-            </button>
+    <div class="flex flex-col gap-2">
+        {% if PROJECTS_ENABLED %}
+            {% if object.accepted_for_funding and not object.project %}
+                <button class="btn btn-primary btn-block"  hx-get="{% url 'funds:submissions:create_project' pk=object.pk %}" hx-target="#htmx-modal">
+                    {% trans "Create Project" %}
```

Comment:
```suggestion
                    {% trans "Create project" %}
```

Super small but the rest of the buttons only capitalize the first word

Reply:
Thanks, resolved